src/containers/PageBuilder/SectionBuilder/SectionCarousel/SectionCarousel.module.css

```diff
@@ -4,10 +4,16 @@
   display: flex;
   align-items: center;
   scroll-behavior: smooth;
+  padding: 64px 0 0 0;
 
-  @media (--viewportMedium) {
+  &:hover .carouselArrows {
+    opacity: 1;
+  }
+
+  /* smartphones, touchscreens: we don't need to show arrows */
```

Comment:
@Gnito how will this work with devices that have a keyboard, mouse/trackpad and a touch screen? Some laptops have touch screens. Latest iPads are also going to this direction with their Magic Keyboards etc.

Reply:
@jannekoivistoinen Well, it's true that this follows the primary input mechanism. So, this needs a bit more testing, but to me, it feels like hover effects on top of the content in this particular scenario are pretty annoying on touch devices. Maybe more annoying than the benefit of showing the arrows if there are secondary pointing inputs that support hover. 

Note: 
since it's obvious from the design that there is more content, a user is likely to try dragging the scrollable pane anyway - which should work. Another way that this should work is by clicking the scrollable pane and then using the arrow keys.

But good question! Maybe I should have used "any-hover: none". (I wasn't even aware of it.)